charts/sscs-tribunals-api/values.ccd.preview.template.yaml

```diff
@@ -47,10 +47,35 @@ java:
     SUBSCRIPTION_NAME: "${SERVICE_NAME}-servicebus-main-topic"
     AMQP_HOST: "sscs-sb-preview.servicebus.windows.net"
     AMQP_USERNAME: "RootManageSharedAccessKey"
+    HEARINGS_EVENT_QUEUE_NAME: ${SERVICE_NAME}-servicebus-tribunals-to-hearing-api
   secrets:
+    HEARINGS_EVENT_SERVICE_BUS_CONNECTION_STRING:
+      secretRef: sscs-sb-preview
+      key: connectionString
     AMQP_PASSWORD:
       secretRef: sscs-sb-preview
       key: primaryKey
+  postgresql:
+    enabled: true
+    primary:
+      persistence:
+        enabled: false
+      initdb:
+        scripts:
+          init.sql: |
+            CREATE DATABASE "data-store" WITH OWNER = hmcts ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
```

Comment:
are we actually using any of these local dbs?

Reply:
I can see these DBs are being used in values.ccd.preview: data-store, definition-store, user-profile, role_assignment, sscsjobscheduler
Unsure of draftstore, evidence, and annotation. hmc-cft-hearing-service can be removed.